aspnetcore/signalr/streaming.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Use streaming in ASP.NET Core SignalR
+author: rachelappel
+description: 
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 06/07/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/streaming
+---
+
+# Use streaming in ASP.NET Core SignalR
+
+By [Brennan Conroy](https://github.com/BrennanConroy)
+
+Streaming is a way for clients to subscribe to a stream of data from the server on demand.
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/live/aspnetcore/signalr/streaming/sample) ([how to download](xref:tutorials/index#how-to-download-a-sample))
+
+## Set up the hub
+
+A hub method automatically becomes a streaming hub method when it returns a variant of `ChannelReader`. For example, return values of `ChannelReader<int>` and `Task<ChannelReader<string>>` are streaming methods. Streaming is useful for scenarios when you don't have all the data to send to the client yet, but will be getting it over time and would like to update the client whenever new data is available. Below is a sample that shows the basics of streaming data to the client. Everytime data is written to the `ChannelReader` the client will be notified of the new data and at the end the `ChannelReader` is completed to tell the client that the stream is closed.
```

Comment:
will be getting -> get 
would like -> want
the client will be notified - >the client is notified
Everytime data is written to the `ChannelReader` the client will be notified of the new data and at the end the `ChannelReader` is completed to tell the client that the stream is closed. 
->
Everytime data is written to the `ChannelReader` the client is notified of the new data. At the end, the `ChannelReader` is completed to tell the client the stream is closed.